install.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/sh -e
 
+SUDO=`which sudo`
+
+if [ "`find "$SUDO" -perm -4000`" = "$SUDO" ]; then
```

Comment:
Actually, `command -v sudo` is usually the way to check if you can execute something.